SPHINXsys/src/shared/bodies/base_body.cpp

```diff
@@ -74,13 +72,13 @@ namespace SPH
 		newly_updated_ = false;
 	}
 	//=================================================================================================//
-	void SPHBody::writeParticlesToVtpFile(std::ostream &output_file)
```

Comment:
It seems using ostream leads to compile failue.  Let me have a look again.

Reply:
Basically, i think that we do not need writeParticlesToVtuFile,  writeParticlesToVtpFile is sufficient.
I use std::ofstream because I am able to compile the lib in windows. Actually I am open to any usage if the lib can be compiled both in windows and linux.

Reply:
ok, I haven't compiled it to windows until now. If supporting windows is a requirement then we should get the CI workflow to reflect it to catch this kind of problem.